config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -71,6 +71,38 @@ postsubmits:
       - name: creds
         secret:
           secretName: pusher-service-account
+  kubernetes/k8s.io:
+  - name: cip-prodrun
+    cluster: test-infra-trusted
+    decorate: true
+    run_if_changed: ".*/manifest.yaml"
+    skip_report: false
+    # Never run more than 1 job at a time. This is because we don't want to run
+    # into a case where an older manifest PR merge gets run last (after a newer
+    # one).
+    max_concurrency: 1
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:876bfc74d070f2eeb94625780b78e757834e1854ec30bc539cfb7553274905d9
+        command:
+        - multirun.sh
+        args:
+        - /app/cip-docker-image.binary
+        - k8s-staging-cluster-api/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+        - k8s-staging-coredns/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+        - k8s-staging-csi/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+        imagePullPolicy: Always
```

Comment:
This doesn't seem super necessary if you're pulling an explicit sha as opposed to a dynamic tag